UnitTestFancyZonesEditor/UnitTestFancyZonesEditor.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0-windows10.0.20348.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.19041.0</TargetPlatformMinVersion>
+    <SupportedOSPlatformVersion>10.0.19041.0</SupportedOSPlatformVersion>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <IsPackable>false</IsPackable>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CsWinRT" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="MSTest.TestAdapter" />
+    <PackageReference Include="MSTest.TestFramework" />
+    <PackageReference Include="coverlet.collector" />
```

Comment:
https://github.com/coverlet-coverage/coverlet

Reply:
this is also causing the build to fail due to the notice file

Reply:
So we should remove the coverlet-coverage package from the test?

Reply:
If we don't need it, don't include it  :)

I have a powershell script verify all nuget packages we leverage in the notice.md.  If anything changes (version, a new one, one removed), it blocks the PR until the change is reflected.  Too easy of a thing for us to miss and 100% can be caught via CI

Reply:
Gotcha makes sense. Thanks for the quick responses!